mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -60,6 +61,11 @@ void CGameModeBase::OnPlayerSpawn(CMomentumPlayer *pPlayer)
 {
 #ifdef GAME_DLL
     pPlayer->SetAutoBhopEnabled(PlayerHasAutoBhop());
+
+    if (pPlayer->m_iButtonsToggled & IN_DUCK)
+        pPlayer->m_iButtonsToggled &= ~IN_DUCK;
+    if (pPlayer->m_iButtonsToggled & IN_JUMP)
+        pPlayer->m_iButtonsToggled &= ~IN_JUMP;
```

Comment:
All toggled buttons should just reset on player spawn, regardless of mode. They should also reset upon mom_restart commands